src/transformers/convert_slow_tokenizer.py

```diff
@@ -1635,14 +1646,16 @@ def converted(self) -> Tokenizer:
 }
 
 
-def convert_slow_tokenizer(transformer_tokenizer) -> Tokenizer:
+def convert_slow_tokenizer(transformer_tokenizer, tiktoken=False) -> Tokenizer:
     """
     Utilities to convert a slow tokenizer instance in a fast tokenizer instance.
 
     Args:
         transformer_tokenizer ([`~tokenization_utils_base.PreTrainedTokenizer`]):
             Instance of a slow tokenizer to convert in the backend tokenizer for
             [`~tokenization_utils_base.PreTrainedTokenizerFast`].
+       tiktoken (bool, optional): Whether to use the `tiktoken` library to convert the tokenizer.
+            Defaults to False.
```

Comment:
```suggestion
       tiktoken (bool, optional): Whether to use the `tiktoken` library to convert the tokenizer instead of sentencepiece.
            Defaults to False.
```